src/status_im2/contexts/chat/messages/content/view.cljs

```diff
@@ -154,7 +154,7 @@
                [audio/audio-message message-data context]
 
                constants/content-type-image
-               [image/image-message 0 message-data context on-long-press]
+               [image/image-message 0 message-data context]
 
                constants/content-type-album
                [album/album-message message-data context on-long-press]
```

Comment:
Do we need to apply this fix for album type too?

Reply:
Hi @smohamedjavid, Thank you very much for reviewing PR.

>Do we need to apply this fix for album type too?

We probably need to refactored these files for consistency, but album is already have both `context` and `long-press` as input
![image](https://github.com/status-im/status-mobile/assets/17097240/9763fee1-4e96-4b7c-ac2a-2b3707c6479a)